frontend/src/index.js

```diff
@@ -29,11 +29,11 @@ const store = createStore(
     )
 
 ReactDOM.render(
+  <Provider store={store}>
     <BrowserRouter>
-        <Provider store={store}>
-            <App />
-        </Provider>
-    </BrowserRouter>,
-    document.getElementById('root')
+      <App />
+    </BrowserRouter>
+  </Provider>,
+  document.getElementById('root')
```

Comment:
Not sure if putting Provider as not the parent changes anything but in the documentation seems to want it as the parent. https://redux.js.org/advanced/usage-with-react-router#connecting-react-router-with-redux-app